client/src/redux/show/showActions.js

```diff
@@ -133,6 +134,8 @@ export const searchShow = (
   onlyDisplayShowsWithNoHost = false,
 ) => async dispatch => {
   try {
+    dispatch({ type: SHOW_FETCHING });
```

Comment:
@jberns Also, does this work for you? I've added this into the `searchShow` action so that `state.show.fetching` is set to `true` when a show search is going on.